src/ErrorHandler.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Jobcloud\KafkaSchemaRegistryClient;
+
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\BackendDatastoreException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\ClientException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\CompatibilityException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\InvalidAvroSchemaException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\InvalidVersionException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\OperationTimeoutException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\PathNotFoundException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\RequestForwardException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\SubjectNotFoundException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\UnauthorizedException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\UnprocessableEntityException;
+use Jobcloud\KafkaSchemaRegistryClient\Exceptions\VersionNotFoundException;
+use Jobcloud\KafkaSchemaRegistryClient\Interfaces\ErrorHandlerInterface;
+use PHPUnit\Framework\MockObject\IncompatibleReturnValueException;
+
+class ErrorHandler implements ErrorHandlerInterface
+{
+    /**
+     * @param array $data
+     * @return void
+     * @throws BackendDatastoreException
+     * @throws ClientException
+     * @throws CompatibilityException
+     * @throws InvalidAvroSchemaException
+     * @throws InvalidVersionException
+     * @throws OperationTimeoutException
+     * @throws PathNotFoundException
+     * @throws RequestForwardException
+     * @throws SubjectNotFoundException
+     * @throws UnauthorizedException
+     * @throws UnprocessableEntityException
+     * @throws VersionNotFoundException
+     */
+    public function handleFromResponseData(array $data): void
```

Comment:
i was thinking, maybe it would be nice to pass the response object which is clearer and gives you more power, wdyt?

Reply:
I was thinking too... you remember that I talked about this.
I tried, the only problem is that I will have to extract `parseJsonResponse` from that method to somewhere else.

I will try again something..

Reply:
it's one line of code, i think for the error handler case, we could live with it, to have this one line (without func) in the code ;)